vendor/bat-native-ledger/include/bat/ledger/ledger.h

```diff
@@ -390,9 +398,11 @@ class LEDGER_EXPORT Ledger {
 
   virtual std::string GetWalletPassphrase() const = 0;
 
-  virtual void LinkBraveWallet(
-      const std::string& destination_payment_id,
-      ResultCallback callback) = 0;
+  virtual void LinkBraveWallet(const std::string& destination_payment_id,
+                               PostSuggestionsClaimCallback callback) = 0;
+
+  virtual void GetDrainID(const std::string& drain_id,
```

Comment:
Perhaps `GetDrainID` → `GetDrainStatus`? 🙂

Reply:
@kylehickinson are you planning on storing the drain ID on the iOS side so that you can poll this method?

Reply:
@zenparsing I believe it was the plan to store the drain ID on the client yeah. Wasn't aware that the ledger side of it was going to store the ID. If it is, then I guess I don't see any reason to pass the `drain_id` into the status method

Reply:
changed the name and removed the drain_id from the callback.